app/src/androidTest/java/com/epfl/drawyourpath/userProfile/cache/UserModelCachedTest.kt

```diff
@@ -103,7 +102,7 @@ class UserModelCachedTest {
         waitUntilAllThreadAreDone()
         // check that it is the correct user
         assertEqualUser(testUserModel, user.getUser().getOrAwaitValue())
-        assertEqualRun(testUserModel.runs!!, user.getRunHistory().getOrAwaitValue())
+        // assertEqualRun(testUserModel.runs!!, user.getRunHistory().getOrAwaitValue())
         // set non working database

```

Comment:
This assertion is commented, maybe add a TODO if you have to uncomment it in a future task, otherwise you can remove it.